account_invoice_report_due_list/views/account_invoice_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2018 Tecnativa - Carlos Dauden
+     Copyright 2018 Tecnativa - Pedro M. Baeza
+     Copyright 2020 Tecnativa - Víctor Martínez
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="groups_id" eval="[(4, ref('account.group_account_invoice'))]" />
+        <field name="arch" type="xml">
+            <field name="invoice_date_due" position="before">
```

Comment:
I found a weird formatting issue with this:
![image](https://user-images.githubusercontent.com/38977934/109017723-5ad20600-76af-11eb-879b-3d029a0cda82.png)
Maybe with_
```suggestion
            <field name="invoice_payment_term_id" position="after">
```
it could work?

Reply:
Still getting the same issue with the suggested change in the runbot :/

![image](https://user-images.githubusercontent.com/38977934/109513408-fa6f0a00-7a9c-11eb-893a-4799e24edf35.png)

To reproduce: 
1. Payment term "30% Now, Balance 60 Days" has 2 term lines
2. Create a customer invoice
3. Select "30% Now, Balance 60 Days" in the "Due date" field
4. Save a see the result

Reply:
Ok, thanks, now `invoice_date_due` field hide when exist multi date due to prevent appear some dates together.

Reply:
Ok, thanks, the issue is fixed now. The only issue I can see now is that, after the invoice has been crated, you can't change from a payment term back to a fixed due date. IDK if that can be a use case, but it could be a problem.

Reply:
If everithing work fine it's correct because only can change invoice date due when payment term haven't multi-lines (when payment term have multi-lines information about date due showed in invoice it's readonly).